### DISCUSSION TITLE
Share my compact layout with custom.css

### DISCUSSION THREAD

**Discussion by Honghe** [Zettlr/Zettlr#4522] [Ideas]
With the compact layout, we can read more contents in one page.

Original layout:
![2023-07-29_18-44_1](https://github.com/Zettlr/Zettlr/assets/1092722/ae602047-125e-42b1-b2b2-ab68755cc6bc)

Compact layout with `custom.css`:
![2023-07-29_18-44](https://github.com/Zettlr/Zettlr/assets/1092722/6172f9c1-554a-4255-89a8-403abcb38c9b)

custom.css:
```
.cm-line {
  margin: -0.2rem 0;
}

.tree-item-container {
  margin: -0.3rem 0;
}

.cm-editor .size-header-1 {
  font-size: 1.25rem;
}

.cm-header-1 {
  font-size: 1.25rem;
}
.cm-header-2 {
  font-size: 1.2rem;
}

.cm-header-3 {
  font-size: 1.15rem;
}

.cm-header-4 {
  font-size: 1.1rem;
}

.cm-header-5 {
  font-size: 1.05rem;
}

.toc-entry-container {
  margin-left: 0 !important;
  margin: -0.3rem 0;
}
```

**Comment by exanrauzer** [comment]
More content like this!
